components/navbar/navbar.css

```diff
@@ -0,0 +1,37 @@
+.navbar-container{
+    width: 100%;
+    display:flex;
+    flex-direction: row;
```

Comment:
flex-direction is by default set to row so you can remove this property

Reply:
yes will do